### PR TITLE
Caja-based sanitizer for AMP templates

### PR DESCRIPTION
**Pull request by dvoytenko**
Related to #657 and #892

### CODE REVIEW

**Comment by cramforce** [comment]
LGTM pending sec review

File: src/sanitizer.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    return false;
+  }
+
+  // Inline styles are not allowed.
+  if (attrName == 'style') {
+    return false;
+  }
+
+  // No attributes with "javascript" in them.
+  if (attrValue.indexOf('javascript') != -1) {
```
Comment (erwinmombay) [inline src/sanitizer.js]: haven't really looked into the code so this is probably a stupid question, would this trigger on a `class="no-javascript"` ?

Comment (dvoytenko) [inline src/sanitizer.js]: Yeah. It would. I will add a colon to it.

Comment (erwinmombay) [inline src/sanitizer.js]: :+1:

Comment (dvoytenko) [inline src/sanitizer.js]: Done

**Pull request merged**